frontend/src/App.js

```diff
@@ -18,8 +19,9 @@ function App() {
                         <Route path="/login" element={<LoginPage/>}/>
                         <Route path="/home" element={<HomePage/>}/>
                         <Route path="/settings" element={<SettingsPage/>}/>
-                        <Route path="/search" element={<SearchPage/>}/>
-                        <Route path="/interview" element={<InterviewPage/>}/>
+                        <Route path="/search" element={<SearchPage />} />
+                        <Route path="/interview" element={<InterviewPage />} />
+                        <Route path="/history" element={<HistoryPage />} />

```

Comment:
Recommend to follow style guide https://arcticicestudio.github.io/styleguide-javascript/rules/react/whitespace.html

So in this case all the components would be `<SomeName />` instead of `<SomeName/>`

Reply:
Will edit for the rest the routes using the way I did it